### PR TITLE
Fix #967: resolved unescaped apostrophe in string resource files

### LINKED ISSUE #967: Resource compilation failed while setting up
**Summary:**

Compilation failed while running the app on device with the following error.

Android resource compilation failed
Output:  /Users/Nikhil/AndroidStudioProjects/mifos-mobile/app/build/intermediates/incremental/mergeDebugResources/merged.dir/values-fr/values-fr.xml:6: error: unescaped apostrophe in string
"Pour afficher les transactions dans votre compte d'épargne, allez à la section Comptes, cliquez sur le compte d'épargne requis, cliquez sur les trois points présents dans le coin supérieur droit et sélectionnez l'option Transaction.".
/Users/Nikhil/AndroidStudioProjects/mifos-mobile/app/build/intermediates/incremental/mergeDebugResources/merged.dir/values-fr/values-fr.xml:6: error: could not parse array item.
/Users/Nikhil/AndroidStudioProjects/mifos-mobile/app/build/intermediates/incremental/mergeDebugResources/merged.dir/values-fr/values-fr.xml:8: error: unescaped apostrophe in string
"Pour créer un héritier, allez à l'héritier sur la page principale de l'application, puis cliquez sur le bouton dans le coin inférieur droit, sélectionnez l'option de numérisation qui ouvrira la caméra de l'appareil, scannez le code QR de la personne pour laquelle vous souhaitez créer un héritier, après avoir rempli les données requises, créer des héritiers en utilisant le code QR".
/Users/Nikhil/AndroidStudioProjects/mifos-mobile/app/build/intermediates/incremental/mergeDebugResources/merged.dir/values-fr/values-fr.xml:8: error: could not parse array item.
/Users/Nikhil/AndroidStudioProjects/mifos-mobile/app/build/intermediates/incremental/mergeDebugResources/merged.dir/values-fr/values-fr.xml:9: error: unescaped apostrophe in string
...........

**Steps to reproduce:**

Setting up the app on a device/emulator.

**Device and Android version:**

Redmi Note 3 API 23

### PR DESCRIPTION
**Pull request by nikhil-sachdeva**
Fixes #967

Resolved unescaped apostrophes in fr,pl,in strings.xml files. The app wasn't running on my device before this and now it's running.

### CODE REVIEW

**Review by luckyman20 — APPROVED** [review]
@nikhil-sachdeva Good work! This will be merged soon.

**Comment by miPlodder** [comment]
@nikhil-sachdeva Good job, I think fix for this issue was added in another PR which I have merged. Anyway, you try to rebase and fix the conflict and see if this still persists or not.

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -385,8 +385,9 @@
     </string-array>
 
     <string-array name="faq_ans">
-        <item>Pour demander un compte de crédit, cliquez sur \ "Signaler la demande de prêt \" sur l\'écran d\'accueil.</item>
-        <item>Vous pouvez afficher les informations de votre profil en cliquant sur l’image de profil sur la page principale de l’application.</item>
+
```
Comment (miPlodder) [inline app/src/main/res/values-fr/strings.xml]: Remove this unwanted space.

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -385,8 +385,9 @@
...
+        <item>Pour demander un compte de crédit, cliquez sur \ "Signaler la demande de prêt \" sur l'écran d'accueil.</item>
```
Comment (miPlodder) [inline app/src/main/res/values-fr/strings.xml]: \ " - remove space in between.
l'écran d'accueil - add escape character at both the apostrophe.

**Review by miPlodder — CHANGES_REQUESTED** [review]

**Comment by miPlodder** [comment]
@nikhil-sachdeva Squash your commits.

**Comment by miPlodder** [comment]
@nikhil-sachdeva Fix the conflict.

**Comment by nikhil-sachdeva** [comment]
Every time I fix the conflict on github, two commits are made, I pull the branch to my local repo to squash the commits, pushing it again shows merge conflicts. Help me

**Comment by luckyman20** [comment]
Don't fix the conflicts here, fix them on your local machine.

**Pull request closed without merging**